source/horoscope/pages/landing.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Horoscope App, Team #5</title>
+    <link rel="icon" type="image/x-icon" href="images/star_favicon.ico">
+    <link rel="stylesheet" type="text/css" href="../css/landing.css">
+    <script src="../js/landing.js" type="module"></script>
+    <!-- download the google font Tillana-->
+    <link href="https://fonts.googleapis.com/css2?family=Tillana:wght@600&display=swap" rel="stylesheet">
```

Comment:
Do we wan to introduce a new font or keep to the old ones to keep things consistent?

If we want to introduce a new font we might want to match how they do it already but downloading the font file?